cmapi/cmapi_server/managers/process.py

```diff
@@ -262,6 +262,41 @@ def noop(cls, *args, **kwargs):
         """No operation. TODO: looks like useless."""
         cls.process_dispatcher.noop()
 
+    @classmethod
+    def gracefully_stop_dmlproc(cls) -> None:
+        """Gracefully stop DMLProc using DBRM commands."""
+        logging.info(
+            'Trying to gracefully stop DMLProc using DBRM commands.'
+        )
+        try:
+            with DBRM() as dbrm:
+                dbrm.set_system_state(
+                    ['SS_ROLLBACK', 'SS_SHUTDOWN_PENDING']
+                )
+        except (ConnectionRefusedError, RuntimeError):
+            logging.error(
+                'Cannot set SS_ROLLBACK and SS_SHUTDOWN_PENDING '
```

Comment:
Ideas:
"Cannot set SS_ROLLBACK and SS_SHUTDOWN_PENDING using DBRM while trying to gracefully auto stop DMLProc."
to
"Cannot set SS_ROLLBACK and SS_SHUTDOWN_PENDING via DBRM, graceful auto stop of DMLProc failed. Try a regular stop method"
or
"Graceful auto stop of DMLProc failed: Cannot set SS_ROLLBACK and SS_SHUTDOWN_PENDING using DBRM. Try a regular stop method"

Reply:
@mariadb-AllenHerrera Try or Trying?